app/assets/javascripts/float-kit/addon/lib/float-kit-instance.js

```diff
@@ -231,4 +233,8 @@ export default class FloatKitInstance {
 
     return this.options.untriggers ?? ["click"];
   }
+
+  get shouldTrapPointerDown() {
+    return true;
+  }
```

Comment:
Not sure if I should check for `this.expanded` here in the `FloatKitInstance` parent class since the `expanded` property is defined in the child class.

Reply:
yes that sounds fine to me